fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -58,7 +61,7 @@ def load_data(self, X, y, event=None, **kwargs):
         zero_vec = pd.Series(0.0, self.index)
         i = 0
         for event_val in event_vals:
-            for group in group_vals[:-1]:
+            for group in group_vals[:-1]:  # Leave out last group to avoid linear dependence.
```

Comment:
I have a vague idea about what this means, but it's not completely clear still.

Reply:
It's a continuation of the longer comment above that says we need a linearly independent basis Adding this last group into the basis makes it linearly dependent.

Reply:
Any suggestion on how to rewrite that?

Reply:
slightly adjusted it, but not sure that helps

Reply:
Well, I _think_ that what you mean is that between two sensitive features there's one tradeoff to be made, between three there are two tradeoffs etc. But I'm not sure. The alternative is for determining the 'size' of the grid in multi-D, which comes down to "if you know the sum of n numbers, then the nth is determined by the other n-1"

Reply:
That is how I understand it. Maybe @MiroDudik  has an idea on how to rewrite this comment?

Reply:
I suppose Miro addressed this with his own comment :-)